src/components/Header/Header.style.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+
+const HeaderWrapper = styled.header`
+  display: flex;
+  height: 65px;
+  width: 100%;
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  width: auto;
+  margin-left: ${(props) => props.theme.margin};
```

Comment:
I guess the design is not that good that we have unified spacing :-)

You can leave it in the theme but I would make the name more specific to the header and less specific regarding the usage as margin. Something like `headerLinkSpacing` for example. Or you introduce a `spacing` object inside the theme that contains an item `headerLinks`. Something like that